### PR TITLE
[NUI] Fix CA2000 warning for Window

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
Fix CA2000 warning for Window

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by AchoWang — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/public/Window.cs [hunk 1]
```diff
@@ -1111,11 +1112,14 @@ public void SetAvailableOrientations(List<Window.WindowOrientation> orientations
             {
                 for (int i = 0; i < orientations.Count; i++)
                 {
-                    orientationArray.PushBack(new PropertyValue((int)orientations[i]));
+                    PropertyValue value = new PropertyValue((int)orientations[i]);
+                    orientationArray.PushBack(value);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Window.cs:1116]: is the `PropertyValue value` copied into `orientationArray` when `PushBack()`?
if it is not copied surely, then the `value` is disposed immediately and the value will be deleted.
then `orientationArray` will not have any data.
could you confirm this?

**Review by AdunFang — APPROVED** [review]
It is OK.

**Pull request merged**